docs/INSTALLATION.md

```diff
@@ -4,7 +4,7 @@
 
 The preferred method of installing Groovy is [SDKman](http://sdkman.io/).
 
-Detailed instructions can be found [here](http://sdkman.io/install.html).  Breifly, open a shell and issue:
+Detailed instructions can be found [here](https://sdkman.io/install/).  Briefly, open a shell and issue:
```

Comment:
I think the address should be `https://sdkman.io/install`, without the `/` at the end. 

```suggestion
Detailed instructions can be found [here](https://sdkman.io/install).  Briefly, open a shell and issue:
```